sentry-android/build.gradle.kts

```diff
@@ -68,6 +68,7 @@ android {
 
 dependencies {
     api(project(":sentry-core"))
+    api(project(":sentry-android-ndk"))
```

Comment:
should not it be `implementation` and expose it through java/JNI classes?

Reply:
also, I might do the opposite, `sentry-android-ndk` have `sentry-android` so who don't use NDK doesn't need to carry it's dependencies

Reply:
Discussed offline:

sentry-android depends on:
1. sentry-android-core
2. sentry-android-ndk

sentry-android-core depends on:
1. sentry-core

Reply:
Would it be possible to have `sentry-android-ndk` depend on e.g. `sentry-core` and have the user add either `io.sentry.android:sentry-android` and/or `io.sentry.android:sentry-android-ndk` (whatever the group and artifacts name will be) depending on what she needs?

Reply:
Yes but we need to prioritize easy docs. A single install instruction for basic usage that exposes all features. Then further down the docs (for those 10% that read that far) will be details on the project structure and how they can drop NDK support, some pros and cons etc. Often we print install instructions on flyers and take to conferences, it must be small and include all features when possible.